pybamm/models/base_model.py

```diff
@@ -614,6 +623,32 @@ def check_variables(self):
                     )
                 )
 
+    def info(self, symbol_name):
+        """
+        Provides helpful summary information for a symbol.
+
+        Parameters
+        ----------
+        parameter_name : str
+        """
+
+        div = "-----------------------------------------"
+        symbol = find_symbol_in_model(self, symbol_name)
+
+        if not symbol:
+            return None
+
+        print(div)
+        print(symbol_name, "\n")
+        print(type(symbol))
+
+        if isinstance(symbol, pybamm.FunctionParameter):
+            print("")
+            print("Inputs:")
+            symbol.input_names
```

Comment:
should this be `print("Inputs:\n{}".format(symbol.input_names)`?

Reply:
`symbol.input_names` just prints the input names. Can change this though if you prefer it to return a list of the input names and print here instead.

Reply:
I see, this is a little bit confusing because (as I understand it) a property should always return something, and a function that prints something should be called with `()`.
I would suggest doing this instead:
```python3
@property
def input_names(self):
    return self._input_names # a property should always return something

def print_input_names(self): # call this as .print_input_names() to print
    for inp in self._input_names:
         print(inp)
```

Reply:
yeah, I'm happy when this is changed